extensions/3DTILES_extent/README.md

```diff
@@ -0,0 +1,136 @@
+# 3DTILES_extent
+
+## Contributors
+
+* Sean Lilley, [@lilleyse](https://github.com/lilleyse)
+* Samuel Vargas [@Samulus](https://github.com/Samulus)
+
+## Contents
+
+* [Overview](#overview)
+* [Defining Extents](#defining-extents)
+* [Holes in Extents](#holes-in-extents)
+* [Arc Types](#arc-types)
+* [Coordinate System](#coordinate-system)
+
+## Overview
+
+The extent 3dtiles extension allows the user to annotate the existence of a smaller 2D region(s) (an extent) in a given `tileset.json` via a 2D array of latitude, longitude pairs. This is useful for a variety of scenarios such as: Overlaying high resolution geometry ontop of low level geometry, insetting one tileset into another tileset, or clipping excess geometry inside of a provided tileset.

```

Comment:
Throughout, change `tileset.json` to `tileset JSON`. The rest of the 3D Tiles spec uses this form since the tileset JSON doesn't have to be called `tileset.json`.